.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 **/*.orig
 .DS_Store
 npm-debug.log
+*.tgz
```

Comment:
avoids including a tgz from a previous `$ npm pack` in another run, especially useful when actually publishing 😅